utils/visualization.py

```diff
@@ -97,6 +97,8 @@ def vis(params, input, output, vis_path, sample_num=0, label=None, dataset='', e
     heatmaps = get_key_def('heatmaps', params['visualization'], False)
     grid = get_key_def('grid', params['visualization'], False)
     ignore_index = get_key_def('ignore_index', params['training'], -1)
+    mean = get_key_def('mean', params['training']['normalization'])
+    std = get_key_def('std', params['training']['normalization'])
```

Comment:
Will it cause a bug if these parameters are missing from the yaml config?

Reply:
No, although it is not tested extensively. I may be wrong. If you come across any bugs please report!